index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Karlovy Vary</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
```

Comment:
Can we use CDN and not local files for external sources since it is just a project and I'd rather keep it clean :) @bytesup